conans/test/model/version_ranges_test.py

```diff
@@ -181,6 +181,7 @@ class SayConan(ConanFile):
 
     def root(self, content, update=False):
         processed_profile = ProcessedProfile()
+        self.loader.cached_conanfiles = {}
```

Comment:
Same comment as @jgsogo. It shouldn't be necessary. Why did you add it? Maybe we are missing something.

Reply:
Please check the above comment.